packages/lib/RotatingLogs.test.ts

```diff
@@ -2,41 +2,24 @@ import { writeFile, readdir, remove } from 'fs-extra';
 import { createTempDir, msleep } from './testing/test-utils';
 import RotatingLogs from './RotatingLogs';
 
-const createLogFile = async (dir: string) => {
-	await writeFile(`${dir}/log.txt`, 'some content');
-};
-
 describe('RotatingLogs', () => {
-	test('should exists an log.txt file inside the folder', async () => {
-		const dir = await createTempDir();
-		await createLogFile(dir);
-		const files = await readdir(dir);
+	test('should rename log.txt to log-TIMESTAMP.txt and then delete it after 1ms', async () => {
+		const dir: string = await createTempDir();
```

Comment:
Consider wrapping each test in a `try { ... } finally { remove the temporary directory }` block. Doing this would ensure that temporary directories get deleted even if a test fails.

Reply:
Wait! Nevermind! I think that the temporary directory (might) be deleted by [`afterAllCleanup`](https://github.com/laurent22/joplin/blob/ffc40b27974d540662212eb6a49a15e0a8e6a301/packages/lib/testing/test-utils.ts#L248).

As such, a `try { ... } finally { ... }` might not be necessary -- it might be sufficient to have an `afterAll` that calls `afterAllCleanup` (e.g. as done [here](https://github.com/laurent22/joplin/blob/ffc40b27974d540662212eb6a49a15e0a8e6a301/packages/lib/services/synchronizer/Synchronizer.tools.test.ts#L2)).